Content.Client/Chat/TypingIndicator/TypingIndicatorVisualizerSystem.cs

```diff
@@ -46,15 +46,11 @@ protected override void OnAppearanceChange(EntityUid uid, TypingIndicatorCompone
         if (!layerExists)
             layer = _sprite.LayerMapReserve((uid, args.Sprite), TypingIndicatorLayers.Base);
 
-        _sprite.LayerSetRsi((uid, args.Sprite), layer, proto.SpritePath, proto.TypingState);
-
-        if (component.UseSyntheticVariant  && currentTypingIndicator != new ProtoId<TypingIndicatorPrototype>("paper")) // L5: Synthetic talk sprites
-        {
+        // L5 - synth typing indicators
+        if (component.UseSyntheticVariant)
```

Comment:
This breaks the writing on paper indicator for synths.

Reply:
uaghg meant to get back to that